test/e2e/case9_templates_test.go

```diff
@@ -517,4 +527,298 @@ var _ = Describe("Test policy templates", func() {
 			utils.ListWithTimeout(clientHubDynamic, gvrPolicy, opt, 0, false, defaultTimeoutSeconds)
 		})
 	})
+
+	Describe("Test a custom service account", Ordered, func() {
+		AfterAll(func(ctx context.Context) {
```

Comment:
Do we need to clean case9SATokenYaml?

Reply:
I think the BeforeAll's DeferCleanUp in "Test token issuance and refresh" covers this. Did I miss something?

Reply:
oh no worry I was confused with token-sa